amit/html/Assigment7/personaldetails.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Home Page</title>
+        <style>
+            body {
+                background-image: url('portal.jpg');
+                background-repeat: no-repeat;
+                background-size: cover;
+            }
+            .matter {
+                text-align: center;
+            }
+            button {
+                background-color: blue;
+
+                padding: 8px 8px;
+
+                border: none;
+                cursor: pointer;
+                width: auto;
+            }
+            .links {
+                text-align: center;
+            }
+            h1 {
+                text-align: left;
+            }
+            .submit {
+                text-align: center;
+            }
+        </style>
+    </head>
+    <body>
+        <div>
+            <h1>Job Portal</h1>
+            <div class="links">
+                <a href="home.html"><button>home</button></a>
+                <a href="personaldetails.html"><button>Personal Detailes</button></a>
+                <a href="Aboutus.html"><button>Aboutus</button></a>
+                <a href="Contactus.html"><button>Contactus</button></a>
+                <a href="login.html"><button>Login</button></a>
+            </div>
+        </div>
+        <div class="matter">
+            <h1>Personal Details</h1>
+            <form>
+                <label for="name">First Name</label>
+                <input type="text" id="first name" name="first name" /><br /><br />
```

Comment:
for attribute given to lable and id given to input filed must be same

make this changes in all assesment